sdks/python/apache_beam/io/filebasedsource_test.py

```diff
@@ -348,6 +357,25 @@ def test_read_zlib_pattern(self):
         compression_type=fileio.CompressionTypes.ZLIB))
     assert_that(pcoll, equal_to(lines))
 
+  def test_remaining_split_points_helper(self):
+    # If current record is at last byte this must be the last split point.
+    self.assertEqual(
+        1, FileBasedSource.remaining_split_points_helper(100, 99, False))
+    # Current record is after stop position but we haven't reached the next
+    # split point
+    self.assertEqual(
+        1, FileBasedSource.remaining_split_points_helper(100, 110, False))
+    # After RangeTracker.set_done() is invoked, there should not be any
+    # remaining split points.
+    self.assertEqual(
+        0, FileBasedSource.remaining_split_points_helper(100, 110, True))
+
+    # If we are at a position < (stop_position -1), FileBasedSource cannot
```

Comment:
either `stop_position - 1` or `stop_position-1`, but be consistent on spaces or I think `-1` is a number here.